includes/polling/os/ibm-as400.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+$output = preg_split('/\s+/', $device['sysDescr']);
```

Comment:
Can you use another variable name other than `$output` to avoid conflicts anywhere else. Maybe `$ibmi_output`.